sheets/api.py

```diff
@@ -464,3 +470,21 @@ def renew_sheet_file_watch(sheet_metadata, force=False):
             file_watch.version += 1
         file_watch.save()
         return file_watch, created, True
+
+
+def get_sheet_metadata_from_type(sheet_type):
+    """
+
+    Args:
+        sheet_type (str):
+
+    Returns:
+
+    """
+    if sheet_type == SHEET_TYPE_COUPON_REQUEST:
+        return CouponRequestSheetMetadata()
+    elif sheet_type == SHEET_TYPE_COUPON_ASSIGN:
+        return CouponAssignSheetMetadata()
+    elif sheet_type in {SHEET_TYPE_ENROLL_CHANGE, WORKSHEET_TYPE_REFUND}:
```

Comment:
Do we still need to handle `SHEET_TYPE_DEFERRAL`?

Reply:
Nope, that was a mistake. The refund sheet and the deferral sheet are both worksheets in the same spreadsheet, so `SHEET_TYPE_ENROLL_CHANGE` will be the sheet type for both. I included `WORKSHEET_TYPE_REFUND` to handle legacy requests.